gcloud-java-resourcemanager/src/main/java/com/google/gcloud/spi/DefaultResourceManagerRpc.java

```diff
@@ -107,5 +116,50 @@ public Project replace(Project project) {
       throw translate(ex);
     }
   }
-}
 
+  @Override
+  public Policy getPolicy(String projectId) throws ResourceManagerException {
+    try {
+      return resourceManager.projects()
+          .getIamPolicy(projectId, new GetIamPolicyRequest())
+          .execute();
+    } catch (IOException ex) {
+      ResourceManagerException translated = translate(ex);
+      if (translated.code() == HTTP_FORBIDDEN) {
+        // Service returns permission denied if policy doesn't exist.
+        return null;
+      } else {
+        throw translated;
+      }
+    }
+  }
+
+  @Override
+  public Policy replacePolicy(String projectId, Policy newPolicy) throws ResourceManagerException {
+    try {
+      return resourceManager.projects()
+          .setIamPolicy(projectId, new SetIamPolicyRequest().setPolicy(newPolicy)).execute();
+    } catch (IOException ex) {
+      throw translate(ex);
+    }
+  }
+
+  @Override
+  public List<Boolean> testPermissions(String projectId, List<String> permissions)
+      throws ResourceManagerException {
+    try {
+      TestIamPermissionsResponse response = resourceManager.projects()
+          .testIamPermissions(
+              projectId, new TestIamPermissionsRequest().setPermissions(permissions))
+          .execute();
+      Set<String> permissionsOwned = ImmutableSet.copyOf(response.getPermissions());
```

Comment:
Nice catch, according to the [javadoc](https://developers.google.com/resources/api-libraries/documentation/cloudresourcemanager/v1beta1/java/latest/), yes.  Will fix.